liquibase-dist/src/main/assembly/assembly-bin.xml

```diff
@@ -67,22 +67,11 @@
             <outputDirectory>lib</outputDirectory>
             <includes>
                 <include>org.yaml:snakeyaml:jar:</include>
-                <include>org.slf4j:slf4j-api:jar:</include>
```

Comment:
Because we shipped slf4j before, and because we want to integrate with slf4j configs they have, I think we need to be including these dependencies still rather than having them shaded in.

Reply:
Agreed,